template-parts/content-tabs.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Template part for displaying navigation tabs
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package nightingale-wp
+ */
+
+ // Start unordered list
+ echo '<ul class="c-tabs">';
+
+	 // Start first "Overview" link to parent page
+	 $link = '<li class="c-tabs__item"><a class="c-tabs__link';
+
+	 if( empty($post->post_parent) ) {
+		 // On first arrival at the parent page, the Overview link is current (and therefore inactive)
+		 // and all the other links are to child pages...
+		 $link .= ' is-current';
+		 $post_parent = $post->ID;
+	 }
+	 else {
+		 // ...but, once a child page is opened, the Overview link should point to the parent page
+		 // and all the other links to sibling pages
+		 $post_parent = $post->post_parent;
+	 }
+
+	 // Finish building and displaying Overview tab
+	 $link .= '" href="';
+	 $link .= get_permalink($post_parent);
+	 $link .= '"><span class="c-sprite  c-sprite--home  c-tabs__icon"></span><span class="c-tabs__text">';
+	 $link .= 'Overview';
+	 $link .= '</span></a></li>';
+	 echo $link;
```

Comment:
Should we check if there are other tabs to be output before outputting an Overview tab?

Reply:
Agreed and done.